src/styles/components/classroom-manager.styl

```diff
@@ -0,0 +1,20 @@
+.classroom-manager
+  &__table
+    background-color: white
+    border: solid 2px #DEDEDE
+
+  .table
+    &__caption
+      bottom: -2.25em
+      font-weight: bold
+      position: relative
+      text-align: left
+
+    &__row-header
+      background-color: GREY_5
+      border-top: solid 2px #DEDEDE !important
```

Comment:
Yep, that's what the `!important` is for. I don't like it either and there may be a better way to deal with it.

The naming is reflecting what its color index is in the zoo grommet theme. I added #DEDEDE to the grey color list and it's the fifth hex code. One of them is I think FOREGROUND? Confusing yes? I'd like to go through and do a bit of renaming at some point.